scripts/volume

```diff
@@ -72,13 +72,17 @@ mixer_value() {
     fi
 }
 
-ACTION_1=$(xrescat i3xrocks.action.volume.left "amixer -q -D $MIXER sset $SCONTROL $(mixer_capability) toggle")
-ACTION_2=$(xrescat i3xrocks.action.volume.middle "amixer -q -D $MIXER sset $SCONTROL $(mixer_capability) ${STEP}${UNIT}+")
-ACTION_3=$(xrescat i3xrocks.action.volume.right "amixer -q -D $MIXER sset $SCONTROL $(mixer_capability) ${STEP}${UNIT}-")
+ACTION_1=$(xrescat i3xrocks.action.volume.left "true")
```

Comment:
I must be missing something but it's not clear to me how `ACTION_1` set to `true` causes any action to occur in the eval expression below.  The old code executes the `amixer` command.

Reply:
Oh, I think the intention is to not perform any action by default for left or middle click.

Reply:
Right, it's clear from the linked to diff.  Perfectly clear, thanks for your contribution @twvd!

Reply:
Yeah, true is just ment to be a no-op. The original code had mute on right click and it moved in your change, so I moved it back to the original button.